sgt-back-end/index.js

```diff
@@ -0,0 +1,122 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `SELECT  "gradeId",
+                        "name",
+                        "course",
+                        "score",
+                        "createdAt"
+                    from "grades"`;
+  db.query(sql, (err, res2) => {
+    if (err) {
+      return res.status(500).send({ error: 'database querying failed.' });
+    } else {
+      return res.status(200).send(res2.rows);
+    }
+  });
+});
+
+app.post('/api/grades', function (req, res) {
+  if (req.body.name === undefined || req.body.course === undefined || req.body.score === undefined || Number(req.body.score) < 0 || Number(req.body.score) > 100) {
+    return res.status(400).send({ error: 'name and course is a required field that is equal to a valid value.Score is a required field that is equal to an integer from 0 to 100' });
+  }
+  const sql = `insert into "grades" ("name","course","score")
+                values ('${req.body.name}','${req.body.course}','${req.body.score}')
+                returning *`;
```

Comment:
Please use parameterized queries for any queries with dynamic data:

---

![image](https://user-images.githubusercontent.com/16143771/131607314-eaf7c9d3-65e8-40cd-ba54-fcc4a9a89057.png)